app/models/review.rb

```diff
@@ -0,0 +1,8 @@
+class Review < ApplicationRecord
+  belongs_to :user, foreign_key: :author_id
```

Comment:
may be worth naming this association :author? Then we call review.author instead of .user. Would require another specification of class_name: "User" on line 2